### ISSUE TITLE
Exit fullscreen message is displayed too often

### ISSUE THREAD

**Issue by hubgit** [desktop/desktop#7916]
**Please describe the problem you think should be solved**

I run GitHub Desktop maximised, and since the last update whenever I switch back to the app it displays the "Press [⌃⌘F] to exit fullscreen" message for a few seconds.

![image](https://user-images.githubusercontent.com/14294/60647331-66ffb480-9e35-11e9-973b-d407873c021f.png)

**[Optional] Do you have any potential solutions in mind?**

Only display the message once, when entering fullscreen.

**Comment by tierninho** [comment]
@hubgit Thanks for reaching out. 

We will consider the suggestion but I am unsure if we will change anything as multiple users have gotten stuck in full screen mode and this notification helps in educating those in such situations.

Perhaps the positioning of the [notification](https://github.com/desktop/desktop/blob/development/app/src/ui/window/full-screen-info.tsx) could be moved. 🤔

**Comment by outofambit** [comment]
Yeah i agree with @tierninho here. @hubgit this might seem a little silly, but could you tell us a little bit more about how this is hindering you? is it that it blocks the interface below? that it's a distracting animation, etc? something else? thank you!

**Comment by hubgit** [comment]
The alert is simply annoying - it grabs attention and appears on top of the content for a few seconds.

I've switched to running GitHub Desktop in a non-maximised window, to avoid this.

**Comment by jdm2212** [comment]
I was about to file this same issue: full-screened GH Desktop is now really frustrating to use because what seems like a notification appears every time I switch to it. The notification is attention-grabbing and makes me immediately lose context on whatever I was doing before.

I'm not aware of any other Mac app that exhibits this behavior when full-screened. I use Terminal, IntelliJ, Chrome, VSCode, and Sublime full-screened pretty regularly. None do this.

**Comment by jdm2212** [comment]
A better alternative would be putting a persistent help thing in the corner that says how to escape full-screen, rather than having it pop up as a notification. It's specifically the pop up as notification quality of it that is distracting and makes full-screened GH Desktop painful to use.

**Comment by tierninho** [comment]
Some more context:

Took at a peek at some other applications (some github-owned and some not) and none of them have a notification for fullscreen mode as mentioned above. 

Also, our previous issues around this feature only relate to escaping fullscreen mode in Windows only, as it can very difficult to do so unless you know beforehand that `F11` is the answer.

I suggest we nix the [toast notification](https://github.com/desktop/desktop/blob/development/app/styles/ui/window/_toast-notification.scss) for Mac, and only leave it for Windows (unless we aim to design a better escape hatch for DOS).

**Comment by Sam-Spencer** [comment]
This is an incredibly annoying notification on macOS and exiting fullscreen is a fairly universal behavior. If a macOS user *enters* fullscreen, it can generally be assumed they know how to exit it.

I understand it may be more helpful on Windows, but on macOS it appears *every* time you swipe / switch over (a common UX behavior in macOS not present in Windows AFAIK) to the full screen GitHub window. This means that as I'm using other apps in full screen and checking GitHub for changes, etc. it appears each time and thus becomes a quasi-persistent UI overlay that mostly gets in the way of work.

Is there any update on this? Could a middle ground solution be a way to turn off this alert in the app's preferences?

**Comment by tierninho** [comment]
@Sam-Spencer Thanks for the input. Currently there is no way to dismiss it and unfortunately we do not have an update on it. We are aware of the issue and hope to fit it in the roadmap.

@ampinsk any ideas on how to style this better?

**Comment by ryandanielspmc** [comment]
This happens to me EVERY time I switch back to the GitHub Desktop app. EXTREMELY annoying.

**Comment by th0th** [comment]
This makes fullscreen experience unusable for me. But I see why it exists in the first place, too. How about making it enabled by default but providing a simple checkbox in settings to disable?

**Comment by tiliv** [comment]
This is still unaddressed?  No other app on earth does this every time its focused...

**Comment by Sam-Spencer** [comment]
I've implemented @tierninho's suggestion to make this a preference (which defaults to `true`) in PR #10554. I'm new to React programming and this is a humongous project, but I did my best to keep with the style and spirit of the existing codebase. If anyone here has any feedback / suggestions / comments / concerns on my PR, I would love to hear them!

Hope this address the issue everyone's been facing and that you'll all soon be able to get back to using GitHub in fullscreen 🥳
